### PR TITLE
mtext fixes (WIP)

### PR DESCRIPTION
**Pull request by lindsayplatt**
work in progress to fix issue #286 

suggested solutions @jiwalker-usgs @jread-usgs 
pull `mtext` into different view, so it doesn't impact the others (have view for `side=1` in addition to `side=c(1,2)`)
OR
keep `axes=FALSE` from trickling into `view$window`

### CODE REVIEW

**Comment by lindsayplatt** [comment]
issue #288 stemmed from conversations about `las` in mtext

**Comment by jordansread** [comment]
add tests @lindsaycarr ?

**Comment by lindsayplatt** [comment]
Will get some tests added before merging @jread-usgs

**Comment by lindsayplatt** [comment]
@jread-usgs the tests I wrote are testing for the behavior we want, so those fail. Should I comment them out or let them fail?

**Comment by jordansread** [comment]
Can you fix the code to pass the tests? or is this some other issue that should be broken out into another issue?

**Comment by lindsayplatt** [comment]
Well, I think the test is failing because of our par scoping stuff that we pulled into a different ticket. The first test that fails is because the second `mtext` call has `las` dropped out of it in `set_window`.  

```
gs <- gsplot() %>% 
    points(1,2) %>% 
    mtext(text=c(1,2,3,4), at=c(0.7,0.9,1.1,1.3), cex=0.5, las=2, side=1, line=1) %>% 
    mtext(text=c("yr1", "yr2"), at=c(0.8, 1.2), las=1, side=1, line=3)

  expect_true(names(gs[['view']][[2]]) %in% "las")
  expect_true(names(gs[['view']][[3]]) %in% "las")
```

**Comment by lindsayplatt** [comment]
@jread-usgs forgot to tag you in my reply. Should I keep those tests in and merge?

**Comment by jordansread** [comment]
Hmm..A little hesitant to merge w/ broken tests. I will play around with your PR and look into the scoping issue.

File: R/mtext.R [hunk 1]
```diff
@@ -28,6 +28,24 @@ mtext <- function(object, ...) {
...
+  
+  user_args <- function_args(name=fun.name, package="graphics", side=side, ...)
+  
+  to.gsplot <- list(list(arguments = c(do.call(set_args, c(fun.name, user_args)), axes=FALSE),  
+                         gs.config = list(legend.name = legend.name, side = side,
+                                          par=par_arguments(user_args, def.funs)))) %>% 
+    setNames(fun.name)
+  
+  object <- append(object, to.gsplot)
+  object <-
```
Comment (jordansread) [inline R/mtext.R]: incomplete line here?

**Comment by jordansread** [comment]
Added some suggested changes as a PR to your fork @lindsaycarr

**Pull request merged**